### DISCUSSION TITLE
Program earth rotation data to mount

### DISCUSSION THREAD

**Discussion by iancward54** [mworion/MountWizzard4#248] [Q&A] [answered]
I have tried updating this data by pressing the button.
I get Download IERS data: [finals.data]
and then - No automation enabbled - upload not possible
I  can't see how to enable automation.

**Comment by mworion** [comment, marked as answer]
Hi,
MW4 needs to have actually the 10micron updater installed to automate the run of programming the data to the mount. Unfortunately 10micron only offers this on windows. As I know, they are working on a web based solution on their mount, but this is not yet released.
If you are not on windows or you don't have the 10micron updater installed, MW4 only download and prepares the data, but cannot transfer it automatically to the mount.
Hope this helps.
Michel
